Tools/AP_Periph/can.cpp

```diff
@@ -1083,6 +1089,130 @@ void AP_Periph_FW::can_baro_update(void)
 #endif // HAL_PERIPH_ENABLE_BARO
 }
 
+
+/*
+  update CAN airspeed
+ */
+void AP_Periph_FW::can_airspeed_update(void)
+{
+#ifdef HAL_PERIPH_ENABLE_AIRSPEED
+    if (!airspeed.healthy()) {
+        static uint32_t last_probe_ms;
+        uint32_t now = AP_HAL::millis();
+        if (now - last_probe_ms >= 1000) {
+            last_probe_ms = now;
+            airspeed.init();
```

Comment:
Why recall init? Init is normally responsible for getting the device handling and registering it's callback. Can't this lead to multiple callbacks running at once for the same sensor?